### DISCUSSION TITLE
Upload / Share photos of cooked / made / completed recipes

### DISCUSSION THREAD

**Discussion by epheterson** [mealie-recipes/mealie#1602] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [ X ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [ X ] Reviewed the existing feature requests and my feature has not been requested.
- [ X ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
There's no way for users to declare they've "made" a recipe and share that result with others, or even to include a photo with comments on the recipe.

**(Optional): Suggest A Solution**
It'd be nice if users could upload photos with comments.

Even better, it'd be nice if each user could click a button that they've "Made" a recipe, including the date it was made and any comments about it, along with an optional photo.

A nice parallel is on Thingiverse where users upload 3D models, others can either comment on that model, or share a "Make" including a photo and any specifics about how they made it (i.e. using custom settings, materials, etc...). Example of a "Makes" page [here](https://www.thingiverse.com/thing:215703/makes).

**Additional Information**
_FILL THESE OUT_
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
I wrote up a FR a little while back that has a similar goal: #1451. It's something I definitely want to work on eventually, but I think it might be better suited post-release.

**Reply by epheterson** [reply]
Oh cool that is very similar to what I'm asking for here, maybe we can merge the two. The only thing not mentioned in yours is uploading a picture.

**Comment by fivestones** [comment]
I'd love to have the option to add more pictures of made recipes too

**Comment by michael-genson** [comment]
FYI the "I made this" dialog now allows pictures
